src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -61,6 +80,7 @@ public function __construct($output = null, string $charset = null, int $flags =
         AbstractDumper::__construct($output, $charset, $flags);
         $this->dumpId = 'sf-dump-'.mt_rand();
         $this->displayOptions['fileLinkFormat'] = ini_get('xdebug.file_link_format') ?: get_cfg_var('xdebug.file_link_format');
+        $this->styles = static::$themes['dark'] ?? self::$themes['dark'];
```

Comment:
this will break the CliDumper, which was overriding the `styles` property, as you now have the HTML styles overriding the CLI ones here

Reply:
The `HtmlDumper` extends the `CliDumper`, not the inverse. So I'm not sure how this could break the `CliDumper`? `HtmlDumper` was already overriding the `CliDumper::$styles` before these changes.